src/Console/IndexCommand.php

```diff
@@ -70,6 +70,8 @@ public function handle()
         $bulkQuery = new BulkQuery($this->getBulkSize());
 
         foreach ($data as $item) {
+            $bulkQuery = new BulkQuery($this->getBulkSize());
```

Comment:
Can you change the instantiation outside of the loop to `$bulkQuery = null;` and remove the reset call inside the loop?

Reply:
Should I keep `$bulkQuery = new BulkQuery($this->getBulkSize());` on line 70 for this block

```
        if ($bulkQuery->hasItems()) {
            $this->elasticsearchService->bulk($bulkQuery->toArray());
        }
```
In case of $data is empty array.

Reply:
Actually I would change it so that `$bulkQuery` is overwritten right after the `bulk()` call instead of at the beginning of the loop. IMO it's more readable that way too.